### ISSUE TITLE
Bug with count method and block

### ISSUE THREAD

**Issue by vladra** [bbatsov/rubocop#1945]
```ruby
array.count(&:blank?)
```

it is producing 'Use size instead of count error'.

```ruby
array.count { |i| i.blank? }
```
producing 'Pass &:blank? as an argument to count instead of block'

**Comment by rrosenblum** [comment]
I think this is being taken care of by #1936 (issue #1935).

**Comment by alexdowad** [comment]
Sure enough, no warning is produced on v0.34.2. This can be closed.
